apps/newsletters-ui/src/app/components/EditNewsletterForm.tsx

```diff
@@ -12,15 +18,29 @@ interface Props {
 	originalItem: NewsletterData;
 }
 
+const sleep = async (ms: number) =>
```

Comment:
Tested locally without this and seemed to work - not clear what this is doing for us? The Patch having completed should be suffient to tell us the update has finished

Reply:
I originally put the delay in to test that the loading/waiting state would work when the update wasn't instant. But since the behaviour I was asked to put in on the last PR for this form was to navigate to the list view when the user submits the change, it struck me that there needs to be enough of a delay for the user to see the message that says what is happening.
I'm not sure that navigating to the list view is the best UX, actually. If they stay on the same page, they can just see the messaging that says 'newsletter updated'.